src/ViewMarc.js

```diff
@@ -10,32 +10,46 @@ import {
 
 class ViewMarc extends React.Component {
   static manifest = Object.freeze({
+    query: {},
     marcRecord: {
       type: 'okapi',
       path: 'source-storage/formattedRecords/:{id}?identifier=INSTANCE',
-    }
+      accumulate: true,
+      throwErrors: false,
+    },
   });
 
+  componentDidUpdate() {
+    let { marcRecord } = this.props;
+    const { onClose } = this.props;
+
+    if (!marcRecord) {
+      this.props.mutator.marcRecord.GET()
+        .then(data => { marcRecord = data; })
```

Comment:
What is this line supposed to do? I don't see you using the local `marcRecord` variable anywhere else in this scope. It almost looks like you're trying to change the value of `this.props.marcRecord`?

Reply:
Yeah, my fault, fixed.